Paws4Me/Paws4Me/Display All Adoptable Pets/View Models/PetViewModel.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  PetViewModel.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/24.
+//
+
+import Foundation
+import UIKit
```

Comment:
Don't think you need this.

Reply:
I now do not because I removed the tableview from the viewModel 👍🏻